apps/site/src/views/Resources/sections/Landing/Landing.tsx

```diff
@@ -1,9 +1,12 @@
+import Image from "next/image";
+
 import styles from "./Landing.module.scss";
+import resourcesTitle from "@/assets/images/resources-title.svg";
 
 function Landing() {
 	return (
 		<div className={styles.landing}>
-			<h1>Resources</h1>
+			<Image src={resourcesTitle} alt="resources title" />
```

Comment:
Issue: the size of the Resources Title stays the same for all screen sizes, so on small screens (i.e iPhone SE), the title starts getting cut off on the sides.